go/internal/registry/registry.go

```diff
@@ -35,44 +35,46 @@ import (
 
 // The global registry, used in non-test code.
 // A test may create their own registries to avoid conflicting with other tests.
-var globalRegistry *registry
+var Global *Registry
```

Comment:
Given this is a var, does it make sense to keep it unexported and export a `GlobalReg()` function instead so that the global registry can't be overwritten/nilled out?

Reply:
Good question.

I would do that if it was user-visible. But it's in an internal package, meaning only code in or under the `go` directory can use it. It's part of the culture to allow this level of "unsafety"; we wouldn't do that in Java but in Go I feel it's okay. It is a judgment call though. Maybe Ian or Keith will disagree.

Reply:
Exported variables in internal packages seems fine.
The only thing I worry about is if there is a protocol involved in accessing it, in which case it should be unexported and have accessor functions. Typically that would be if we need to hold a lock around the access, or use atomic read/write, or whatever.
If it is ok to just read this variable whenever (which I think it is, given it is set in init?), then exported is fine.